.babelrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
@mini-eggs What do we need babel for?

Reply:
Not sure how this got in here... Must have been auto gen'd by something. I was playing with Parcel while dev'ing -- maybe that snuck it in?